sale_order_revision/view/sale_order.xml

```diff
@@ -16,7 +16,7 @@
         <field name="inherit_id" ref="sale.view_order_form" />
         <field name="priority">15</field>
         <field name="arch" type="xml">
-            <xpath expr="//header" postion='inside'>
+            <xpath expr="//header" position='inside'>
```

Comment:
@CristianoMafraJunior I don't see it on runboat:

![image](https://github.com/OCA/sale-workflow/assets/19529533/ecb94153-7167-433b-a048-b0d878191e49)

Reply:
![primeiraimagem](https://github.com/OCA/sale-workflow/assets/142639425/fd009ed8-e094-4622-8d7b-756268667a38)
![segundaimagem](https://github.com/OCA/sale-workflow/assets/142639425/14853117-9b3f-4ae9-905e-a49beb64f302)

This button is associated with an action called create_revision and must be available when the sales order status is 'cancel' or 'sent'.

Reply:
https://github.com/OCA/sale-workflow/assets/142639425/56c0c526-d6ed-43ac-adf8-920471677150

As demonstrated here in the video, when clicking on the convert to order button, the "New Revision of Quotation" button will appear and when the email is sent, the "New Revision of Quotation" button will also appear